zipkin-cassandra-core/src/main/resources/cassandra-schema-cql3.txt

```diff
@@ -51,3 +51,15 @@ CREATE TABLE IF NOT EXISTS zipkin.traces (
     span      blob,
     PRIMARY KEY (trace_id, ts, span_name)
 ) WITH compaction = {'class': 'org.apache.cassandra.db.compaction.DateTieredCompactionStrategy', 'max_sstable_age_days': '1'};
+
+// This table uses bigint to store timestamp in microseconds, as Cassandra's timestamp is apparently limited to milliseconds
+CREATE TABLE IF NOT EXISTS zipkin.span_duration_index (
+    s      text,      // service name
```

Comment:
all of these keys are different than the prior ones in the same file, which could lead to confusion. I'd suggest the following, if this is ideal.

1. add a line noting why single-character preference is used, as it obfuscates the schema. In other words, make it obvious why we are being cryptic (ex for some performance reason?)
2. add a TODO: to change the other schema to the same naming convention.